roc/version.go

```diff
@@ -0,0 +1,52 @@
+package roc
+
+/*
+#include <roc/version.h>
+*/
+import "C"
+
+import (
+	"runtime/debug"
+	"sync"
+)
+
+var (
+	version     *Version
+	versionOnce sync.Once
+)
+
+// Version components.
+type Version struct {
+	Library *SemanticVersion // Native library version, librock version.
+	Binding string           // Binding version.
```

Comment:
* Can we parse string into components and use SemanticVersion for this field too?
* Binding => Go bindings